tslint.json

```diff
@@ -4,6 +4,7 @@
   ],
   "rules": {
     "arrow-parens": false,
+    "ban-types": false,
```

Comment:
If we made the return type more specific, we wouldn't need to change this here, right?

Reply:
`ban-types` by default will ban return types of `Function`. Since it wasn't used anywhere else, I turned off the rule. 

If you don't like turning off that tslint rule for the tsoa project, another option would be to change the return types from Function to something like DecoratorReturn, and then in another file, define decorator return like this:

```
export type DecoratorReturn = () => void;
```

Reply:
Yeah, I think that would be nicer.

Reply:
ok, I'll add that. I'll put the DecoratorReturn definition in the src/interfaces/ folder.